### PR TITLE
[Feature] Gurkify

### PR DESCRIPTION
**Pull request by Arnav-2004**
This pr will close issue #35

### CODE REVIEW

**Review by JacobMonck — APPROVED** [review]
Looks good to me.

**Review by DudeBro249 — APPROVED** [review]
Looking good 😎!

File: bot/exts/gurkan/gurkify.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+from discord import Embed
+from discord.ext import commands
+
+
+class Gurkify(commands.Cog):
+    """Cog for the gurkify command."""
+
+    @commands.command(name="gurkify")
+    async def gurkify(self, ctx: commands.Context) -> None:
+        """Gurkifying users display name."""
```
Comment (RohanJnr) [inline bot/exts/gurkan/gurkify.py:14]:
```suggestion
        """Gurkify user's display name."""
```

File: bot/exts/gurkan/gurkify.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            embed = Embed(
+                title=random.choice(NEGATIVE_REPLIES),
+                description=(
+                    "Your nick name is too long to be gurkified. "
+                    "Please change it to be under 26 characters."
+                ),
+                color=Colours.soft_red,
+            )
+            await ctx.send(embed=embed)
+            return
```
Comment (RohanJnr) [inline bot/exts/gurkan/gurkify.py:27]:
```suggestion
            return
```
```suggestion
```

**Review by RohanJnr — CHANGES_REQUESTED** [review]

**Comment by gustavwilliam** [comment]
@Arnav-2004 I did some error handling on this, so we might get it merged soon. Some refactoring would be nice for the error embeds. I might do it now.

**Review by gustavwilliam — APPROVED** [review]
Let's try it out

**Pull request merged**